_templates/page.html.erb

```diff
@@ -1,97 +1,24 @@
-  <script src="<%= File.join(javascripts_path, "bootstrap-offcanvas.js") %>" type="text/javascript"></script>
-  <%= render("_templates/_analytics.html.erb", :distro_key => distro_key) %>
-</head>
-<body>
-  <%= render("_templates/_topnav.html.erb", :distro_key => distro_key) %>
-  <div class="container">
-    <p class="toggle-nav visible-xs pull-left">
-      <button class="btn btn-default btn-sm" type="button" data-toggle="offcanvas">Toggle nav</button>
-    </p>
-    <ol class="breadcrumb">
-      <li class="sitename">
-        <a href="<%= site_home_path %>"><%= site_name %></a>
-      </li>
-      <li class="hidden-xs">
-        <%= breadcrumb_root %>
-      </li>
-      <li class="hidden-xs">
-        <%= breadcrumb_group %>
-      </li>
-      <%= breadcrumb_subgroup_block %>
-      <li class="hidden-xs active">
-        <%= breadcrumb_topic %>
-      </li>
-    </ol>
-    <div class="row row-offcanvas row-offcanvas-left">
-      <div class="col-xs-8 col-sm-3 col-md-3 sidebar sidebar-offcanvas">
-        <div class="row-fluid">
-          <%= render("_templates/_search.html.erb", :distro_key => distro_key) %>
-        </div>
-        <%= render("_templates/_nav.html.erb", :navigation => navigation, :group_id => group_id, :topic_id => topic_id, :subgroup_id => subgroup_id, :subtopic_shim => subtopic_shim) %>
-      </div>
-      <div class="col-xs-12 col-sm-9 col-md-9 main">
-        <div class="page-header">
-          <h1><%= article_title %></h1>
-        </div>
-        <%= content %>
-      </div>
-    </div>
-    <div class="visible-xs-block row row-offcanvas row-offcanvas-left">
-      <%= render("_templates/_footer.html.erb", :distro_key => distro_key, :images_path => images_path) %>
-    </div>
-  </div>
-  <script type="text/javascript">
-    /*<![CDATA[*/
-    $(document).ready(function() {
-      $("[id^='topicGroup']").on('show.bs.collapse', function (event) {
-        if (!($(event.target).attr('id').match(/^topicSubGroup/))) {
-          $(this).parent().find("[id^='tgSpan']").toggleClass("fa-angle-right fa-angle-down");
-        }
-      });
-      $("[id^='topicGroup']").on('hide.bs.collapse', function (event) {
-        if (!($(event.target).attr('id').match(/^topicSubGroup/))) {
-          $(this).parent().find("[id^='tgSpan']").toggleClass("fa-angle-right fa-angle-down");
-        }
-      });
-      $("[id^='topicSubGroup']").on('show.bs.collapse', function () {
-        $(this).parent().find("[id^='sgSpan']").toggleClass("fa-caret-right fa-caret-down");
-      });
-      $("[id^='topicSubGroup']").on('hide.bs.collapse', function () {
-        $(this).parent().find("[id^='sgSpan']").toggleClass("fa-caret-right fa-caret-down");
-      });
-    });
-    /*]]>*/
-  </script>
-  <%= render("_templates/_footer.html.erb", :distro_key => distro_key, :images_path => images_path) %>
-</body>
-</html>
+<%= render((distro_key['atomic'] ? "_templates/_page_atomic.html.erb" : "_templates/_page_atomic.html.erb"),
```

Comment:
So, I added some debugging to the asciibinder gem, and narrowed it down to the footer...still trying to understand why that is causing problems though.

```
/Users/jfrey/dev/openshift-docs/_templates/_footer.html.erb:1:in `block in singleton class': undefined local variable or method `images_path' for #<AsciiBinder::TemplateRenderer:0x007f8b2c0b3ad8> (NameError)
	from /Users/jfrey/dev/openshift-docs/_templates/_footer.html.erb:-6:in `instance_eval'
	from /Users/jfrey/dev/openshift-docs/_templates/_footer.html.erb:-6:in `singleton class'
	from /Users/jfrey/dev/openshift-docs/_templates/_footer.html.erb:-8:in `__tilt_70117841297920'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:164:in `call'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:164:in `evaluate'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:99:in `render'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/template_renderer.rb:24:in `render'
	from /Users/jfrey/dev/openshift-docs/_templates/_page_openshift.html.erb:92:in `block in singleton class'
	from /Users/jfrey/dev/openshift-docs/_templates/_page_openshift.html.erb:-28:in `instance_eval'
	from /Users/jfrey/dev/openshift-docs/_templates/_page_openshift.html.erb:-28:in `singleton class'
	from /Users/jfrey/dev/openshift-docs/_templates/_page_openshift.html.erb:-30:in `__tilt_70117841297920'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:164:in `call'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:164:in `evaluate'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:99:in `render'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/template_renderer.rb:24:in `render'
	from /Users/jfrey/dev/openshift-docs/_templates/page.html.erb:1:in `block in singleton class'
	from /Users/jfrey/dev/openshift-docs/_templates/page.html.erb:-29:in `instance_eval'
	from /Users/jfrey/dev/openshift-docs/_templates/page.html.erb:-29:in `singleton class'
	from /Users/jfrey/dev/openshift-docs/_templates/page.html.erb:-31:in `__tilt_70117841297920'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:164:in `call'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:164:in `evaluate'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/tilt-2.0.2/lib/tilt/template.rb:99:in `render'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/template_renderer.rb:24:in `render'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:259:in `page'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:761:in `configure_and_generate_page'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:601:in `block (4 levels) in generate_docs'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:591:in `each'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:591:in `block (3 levels) in generate_docs'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:587:in `each'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:587:in `block (2 levels) in generate_docs'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:544:in `each'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:544:in `block in generate_docs'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:511:in `each'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/lib/ascii_binder/helpers.rb:511:in `generate_docs'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/bin/asciibinder:14:in `call_generate'
	from /Users/jfrey/.gem/ruby/2.2.4/gems/ascii_binder-0.1.2/bin/asciibinder:268:in `<top (required)>'
	from /Users/jfrey/.gem/ruby/2.2.4/bin/asciibinder:23:in `load'
	from /Users/jfrey/.gem/ruby/2.2.4/bin/asciibinder:23:in `<main>'
```

TODO for me: create a PR on asciibinder to properly expose errors. cc @nhr